index.html

```diff
@@ -6,6 +6,115 @@
     <title>Homework 1</title>
   </head>
   <body>
-    <h1>Start developing here</h1>
+    <header>
+      <div class="header">
+        <h1><b><span style="color: blue;">Web</span>studio</b>
```

Comment:
this is not the main title. This is a link with a text. No need to add styles in this task